src/devices/video/snes_ppu.h

```diff
@@ -314,10 +314,14 @@ class snes_ppu_device :  public device_t,
 	virtual void device_reset() override;
 
 	// device_palette_interface overrides
-	virtual uint32_t palette_entries() const override { return 0x100 + (0x100 * 8) + 1; } // 256 word CG RAM data, 8 group of direct colours, Fixed color
+	// 256 word CG RAM data (0x000-0x0ff), 8 group of direct colours (0x100-0x8ff), Fixed color (0x900)
+	virtual uint32_t palette_entries() const override { return 0x100 + (0x100 * 8) + 1; }
 	virtual uint32_t palette_indirect_entries() const override { return 32 * 32 * 32; } // 15 bit BGR
 
 private:
+	const uint16_t DIRECT_COLOUR = 0x100; // Position in palette entry for direct colour
+	const uint16_t FIXED_COLOUR = 0x100 + (0x100 * 8); // Position in palette entry for fixed colour
```

Comment:
These should be `static constexpr` – you don’t want to create instance members for them.